std_msgs/Header header # stamp based on PPS time
dji_sdk/HardSyncDebug hardsync_timestamp
